### ISSUE TITLE
Not works on Gingerbread (4.1)

### ISSUE THREAD

**Issue by Khristian-kun** [byteandahalf/GenericLauncher#20]
It compiles, but when i open it, insta-crash. How to fix it.

**Comment by Razvan-MCrafter** [comment]
Probably it isn't compiling native code. I have the same problem

**Comment by Khristian-kun** [comment]
maybe... but i need this for make my own launcher

**Comment by jv110** [comment]
But 4.1 is Jelly Bean, what do you mean with that?

**Comment by jv110** [comment]
You need to download native code support.

**Comment by Khristian-kun** [comment]
dont get it? when open crashes, and obviously i have NDK support

**Comment by jv110** [comment]
Then you're not using the correct MCPE version.

Btw, why are you using 4.1? You should be using 7.0/6.1.

**Comment by Khristian-kun** [comment]
because my device doesnt have updates

**Comment by Khristian-kun** [comment]
im using 0.14.0

**Comment by TheBigHacker0100** [comment]
Latest GenericLauncher = 0.10.4

**Comment by derderder77** [comment]
@TheBigHacker0100
Not really, the last update work with 0.14.0 and I have a pull request for 0.15.4 waiting since months.
The project seem to be dead anyway, no activity and response from byteandahalf for quite a long time, and the latest update were just accepted pull request.

**Comment by Khristian-kun** [comment]
@derderder77 anyways, could you update it to lastest version? i really want to develop a launcher

**Comment by derderder77** [comment]
@Khristian-kun I don't have many time at this moment, but I will probably update for 1.0 around the 15 February. 

You can also try to update it yourself: 
Just launch the current game with the latest version of GenericLauncher. Use the 0.15.4 version of my fork, not the 0.14 in this repo. I have started the 0.16 support a few month ago, but don't have finished due to a lack of interest/motivation/time, I will push what I has already made in a new branch tomorrow.

So just launch the launcher with the current MCPE installed, it will crash. Open the logs and fix the problem, probably an unimplemented Class/method. Implement it ( apktool and jd-gui are your best friends ), relaunch your modified launcher and fix the next error until you can launch the game. They will be missing/buggy things in comparaison of the official launcher, but you know how to fix them now :). 

Feel free to do pull request of your work on my repo, even if it's not complete nor even working !

**Comment by Khristian-kun** [comment]
@derderder77 ohh thanks! i'll try to get it working sometime
